### ISSUE TITLE
PiSignage Open-source Server Admin usernamen and password issue

### ISSUE THREAD

**Issue by budakatly** [colloqi/pisignage-server#140]
Hi,

My newly built PiSignage server resets my admin username and password every few days, my groups and registered players are also gone. Except for the playlist and assets they are still fine. Please advise. Many thanks!

**Comment by colloqi** [comment]
Looks like mongodb is not setup properly or not connected. playlists and assets are files and do not use database

**Comment by budakatly** [comment]
I see. How to do I check if the mongodb is setup properly? Newbie here. Thanks!

**Comment by budakatly** [comment]
How do you check if the mongodb is working properly under the docker container? Thanks!

**Comment by eXtreme-bg** [comment]
I have the same issue. Is there a known solution already?

**Comment by colloqi** [comment]
Can you check with the latest update?

**Comment by ebandur-fshark** [comment]
This is happening because your server is getting ransomware. If you use the docker install for open source server it exposes the Mongo port to the outside world which means anyone can access your database without credentials. mongodb://SERVERIP:27017 will allow direct access to all of your pisignage information.

Specifically in the yml file 
ports:
      - 27017:27017

is exposing your database. If you were to download Compass by Mongo and enter the connection string you will see that your database has a file from the ransomware attackers with instructions to decrypt your database.

As far as a fix, you would need to look into if 27017 needs to be open for the outside world as I am not too familiar with docker ports but I did do enough digging to figure out this was indeed ransomware causing this.

TLDR:
Ransomware due to port 27017 being exposed to the outside world without authentication so any attacker can find your server IP and have access to your data.
